mobile/ios/ios-proxy-services.ts

```diff
@@ -301,6 +301,14 @@ export class NotificationProxyClient {
 		}
 	}
 
+	private postNotificationCore(notificationName: string): void {
+		this.plistService.sendMessage({
```

Comment:
is it possible plistService to be null here? For example when calling postNotificationAndAttachForData who will initialize plistService?